source/_patterns/colors/intro.hbs

```diff
@@ -0,0 +1,32 @@
+<main class="colors-documentation">
+    <section>
+        <h1>Color System</h1>
+        <p>DB UX Design Guidelines 3.0 introduce a complete new set of color variables that follow some rules to work with.</p>
+        <p>Each accent color (primary, secondary, information, etc.) is provided in a group of six compatible colors of different tones for pairing, defining emphasis, and visual expression.</p>
+        <p>Background colors define their own foreground color by so named on-colors. For using the DB Design System with SCSS, we provide a wide range of SCSS placeholders and utility 
+            classes that automatically style e.g. text and links based on their current background color. For more information have a look at the usage section.</p>
+    
+        <h2>Color Scheme – Primary (Example)</h2>
+        <p>The following example shows the available colors for the primary brand color. 
+            Each defined color has an interactive mode which means that colors for hover and pressed states are defined.</p>
+        <dl class="DO-NOT-COPY-THIS-CLASS-example-docs-listing">
+            <dt><b>Primary</b></dt><dd>base key color</dd>
+            <dt><b>On Primary</b></dt><dd>is applied to content (icons, text, etc.) that sits on top of primary</dd>
+            <dt><b>Primary Background-Light</b></dt><dd>background color for current key color. Only used for backrounds.</dd>
+            <dt><b>Primary Background Transparent</b></dt><dd>Transparent or semi transparent backgrounds for current key color, e.g. used for form field backgrounds.</dd>
+            <dt><b>On Primary Background</b></dt><dd>is applied to content (icons, text, etc.) that sits on top of primary background colors (background light or transparent).</dd>
+            <dt><b>On Primary Background Weak</b></dt><dd>is applied to content (icons, text, etc.) that sits on top of primary background colors (background light or transparent) and is a variant to weaken content on primary backgrounds.</dd>
```

Comment:
@annsch you could style the `dt` elements in bold via CSS instead of the `b` HTML tags.